be/src/storage/lake/spill_mem_table_sink.h

```diff
@@ -73,8 +73,10 @@ class SpillMemTableSink : public MemTableSink {
     TabletWriter* _writer;
     // destroy spiller before runtime_state
     std::shared_ptr<RuntimeState> _runtime_state;
+    RuntimeProfile _dummy_profile{"dummy"};
```

Comment:
There is a load profile in `LoadChannel`, we can use that.

Reply:
ok